web/app/globals.css

```
@@ -0,0 +1,508 @@
+:root {
+  --metal: #18120d;
+  --paper: #f4efe6;
+  --muted: #c4b8a8;
+  --line: #3a3129;
+  --lime: #d3fd64;
+  --glass: rgba(24, 18, 13, 0.62);
+  --ink: #18120d;
+  --sans: var(--font-sans), "Space Grotesk", "Geist", ui-sans-serif, sans-serif;
+  --mono: var(--font-mono), "JetBrains Mono", ui-monospace, monospace;
+  --display: var(--font-display), "Iowan Old Style", Palatino, serif;
```

Stylelint (17.14.0)

[error] Expected "Palatino" to be "palatino" (value-keyword-case)

(value-keyword-case)